lib/src/debug_logger.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_reactive_ble/src/model/log_level.dart';
+
+class DebugLogger {
+  DebugLogger(void Function(Object object) printToConsole)
+      : _printToConsole = printToConsole;
+
+  void enable() {
+    _logLevel = LogLevel.verbose;
+  }
+
+  void disable() {
+    _logLevel = LogLevel.none;
+  }
```

Comment:
If it's either enabled or disabled, then why to use an enumeration `_logLevel` and not boolean `enabled`?

Reply:
I decided to put an enum because maybe in the future we would like a verbose logging and a less verboselogging, but not sure to be honest

Reply:
So, be consistent: replace the enable/disable methods with `setLogLevel(·)`. Otherwise this internal flexibility cannot be leveraged without changing the public interface of this class, which would be a pity.

Reply:
I agree will do that